server/service/integration_mdm_test.go

```diff
@@ -12918,42 +12927,32 @@ INSERT INTO host_mdm_apple_declarations (
 
 	t.Run("Declaration", func(t *testing.T) {
 		want := noTeamDeclsByUUID["123"]
-		wantBytes, err := json.Marshal(want.Declaration)
-		require.NoError(t, err)
 		r, err := mdmDevice.DeclarativeManagement(fmt.Sprintf("declarations/%s/%s", "configuration", want.Identifier))
 		require.NoError(t, err)
 
-		_ = parseDeclarationResp(r, wantBytes)
+		assertDeclarationResponse(r, want)
 
 		// insert a new declaration
 		noTeamDeclsByUUID["abc"] = fleet.MDMAppleDeclaration{
 			DeclarationUUID: "abc",
 			TeamID:          ptr.Uint(0),
 			Identifier:      "com.example4",
 			Name:            "Example4",
-			DeclarationType: fleet.MDMAppleDeclarativeConfiguration,
-			Declaration: json.RawMessage(`{
+			Category:        fleet.MDMAppleDeclarativeConfiguration,
+			RawJSON: json.RawMessage(`{
 				"Type": "com.apple.configuration.test",
 				"Payload": {"foo":"bar"},
-				"Identifier": "com.example4",
-				"ServerToken": "csumabc"
+				"Identifier": "com.example4"
 			}`),
-			MD5Checksum: "csumabc",
-			CreatedAt:   then.Add(3 * time.Minute),
-			UploadedAt:  then.Add(3 * time.Minute),
+			CreatedAt:  then.Add(3 * time.Minute),
+			UploadedAt: then.Add(3 * time.Minute),
 		}
 		insertDeclaration(t, noTeamDeclsByUUID["abc"])
 		insertHostDeclaration(t, mdmDevice.UUID, noTeamDeclsByUUID["abc"])
 		want = noTeamDeclsByUUID["abc"]
-		wantBytes, err = json.Marshal(want.Declaration)
-		require.NoError(t, err)
 		r, err = mdmDevice.DeclarativeManagement(fmt.Sprintf("declarations/%s/%s", "configuration", want.Identifier))
```

Comment:
Probably need to should fix the "s" here too?

Reply:
sounds good, the test pass because the prefix still matches, but it's good to clean this up.

Reply:
In that case, maybe we should make the prefix test for the slash too: "declaration/"